www/src/components/Menu.jsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link, graphql, useStaticQuery } from 'gatsby';
-import { Container, Row, Col, Tooltip, OverlayTrigger } from '~paragon-react'; // eslint-disable-line
+import Icon from '../../../src/Icon';
```

Comment:
[curious] nit: Should the `Icon` import come from `~paragon-react` below instead of from `src` for consistency?

Reply:
Yes good idea